src/chunkserver/watchdog/dog.cpp

```diff
@@ -0,0 +1,179 @@
+/*
+ *  Copyright (c) 2020 NetEase Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/*
+ * Project: curve
+ * Created Date: 2021-02-03
+ * Author: qinyi
+ */
+
+#include "src/chunkserver/watchdog/dog.h"
+
+#include <mntent.h>
+#include <signal.h>
+
+#include <chrono>
+#include <iostream>
+#include <memory>
+#include <thread>
+
+#include "src/chunkserver/watchdog/watcher.h"
+
+namespace curve {
+namespace chunkserver {
+
+static void* Watch(void* currDog) {
```

Comment:
why not 
static void Watch(void* currDog)

Reply:
done